docs/metrics.md

```diff
@@ -69,6 +114,26 @@ Here's an example of a complete telemetry ping:
 }
 ```
 
+A Redshift schema for the payload:
+
+```js
+local schema = {
+--   column name                   field type   length  attributes   field name
+    {"timestamp",                  "TIMESTAMP", nil,    "SORTKEY",   "Timestamp"},
+    {"uuid",                       "VARCHAR",   36,      nil,         get_uuid},
+
+    {"test",                       "VARCHAR",   255,     nil,         "test"},
+    {"agent",                      "VARCHAR",   45,      nil,         "agent"},
+    {"didNavigate",                "VARCHAR",   56,      nil,         "payload[didNavigate]"},
+    {"interactionType",            "VARCHAR",   200,     nil,         "payload[interactionType]"},
+    {"recommendationShown",        "VARCHAR",   255,     nil,         "payload[recommendationShown]"},
+    {"recommendationType",         "VARCHAR",   255,     nil,         "payload[recommendationType]"},
+    {"recommendationSelected",     "VARCHAR",   36,      nil,         "payload[recommendationSelected]"},
```

Comment:
didNavigate, recommendationShown, recommendationSelected are booleans, so we should use the boolean type for those